### PR TITLE
Iclr19/bidding stage

### PR DESCRIPTION
**Pull request by mohituniyal**

### CODE REVIEW

File: venues/ICLR.cc/2019/Conference/python/invitations.py [hunk 1]
```diff
@@ -236,6 +236,51 @@
     public_comment_template['process'] = f.read()
 
 
+public_comment_template = {
```
Comment (melisabok) [inline venues/ICLR.cc/2019/Conference/python/invitations.py]: it this part of the bids?

File: venues/ICLR.cc/2019/Conference/python/post-submissions-stage.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
 
     client.post_invitation(iclr19.blind_submission_inv)
 
-    original_notes = openreview.tools.iterget_notes(client, invitation=iclr19.submission_inv.id)
```
Comment (melisabok) [inline venues/ICLR.cc/2019/Conference/python/post-submissions-stage.py]: Why did you change this line?

File: venues/ICLR.cc/2019/Conference/python/invitations.py (hunk 1 above)
Comment (mspector) [reply venues/ICLR.cc/2019/Conference/python/invitations.py]: there is already a public comment template, right above this

File: venues/ICLR.cc/2019/Conference/python/post-submissions-stage.py (hunk 2 above)
Comment (mspector) [reply venues/ICLR.cc/2019/Conference/python/post-submissions-stage.py]: I'm guessing this was unintentional, since it breaks the indentation

File: venues/ICLR.cc/2019/Conference/python/iclr19.py [hunk 3]
```diff
@@ -443,6 +443,21 @@
         ],
         'invitees': [],
         'web': os.path.abspath('../webfield/bidWebfield.js')
+    },
+    content_params = {
```
Comment (mspector) [inline venues/ICLR.cc/2019/Conference/python/iclr19.py]: if this is going to be our general solution for other conferences, this content should be changed in openreview-py

Comment (mspector) [reply venues/ICLR.cc/2019/Conference/python/iclr19.py]: see here: https://github.com/iesl/openreview-py/pull/163
